faqs/atm7_http.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to enable HTTP on ATM7
+search: atm7
+---
+
+**Singleplayer**
+1. Press the windows key and R at the same time
```

Comment:
Probably wise to not assume that the user is using a specific operating system.

Reply:
>  if they are using linux i would expect them to know

I wouldn't, while many Linux users are knowledgeable on this kind of thing not all of them are.

Also, they could be using MacOS.